geometry/point.hpp

```diff
@@ -22,23 +22,40 @@ namespace geometry {
 template<typename Vector>
 class Point {
  public:
-  Point();
+  constexpr Point();
 
-  Vector operator-(Point const& from) const;
+#if PRINCIPIA_COMPILER_MSVC && !__INTELLISENSE__
+  // Explicitly define constexpr default copy and move constructors because
+  // otherwise MSVC fails to initialize constant expressions.  In addition,
+  // intellisense gets confused by these (because of course MSVC and
+  // intellisense are different compilers and have bugs in different places).
+  constexpr Point(Point const& other);
+  constexpr Point(Point&& other);
+  Point& operator=(Point const& other) = default;
+  Point& operator=(Point&& other) = default;
+#endif
 
-  Point operator+(Vector const& translation) const;
-  Point operator-(Vector const& translation) const;
+  // TODO(egg): constexpr all the operators.
```

Comment:
Oh, I ended up doing all the member operators.
There are still the non-members, moving the comment.